pkg/services/ngalert/api/tooling/definitions/cortex-ruler.go

```diff
@@ -318,3 +318,7 @@ type GettableGrafanaRule struct {
 	NoDataState     NoDataState         `json:"no_data_state" yaml:"no_data_state"`
 	ExecErrState    ExecutionErrorState `json:"exec_err_state" yaml:"exec_err_state"`
 }
+
+type AdminConfiguration struct {
```

Comment:
How about this? Else it gets confusing with the other `AdminConfiguration` :)
```suggestion
type GettableAdminConfiguration struct {
```

Reply:
This is not only gettable because you also post it, so I'd have to have two types aliasing each other - is this what you're suggesting?

In my mind, they both come from different packages that make it clear what their purpose is for e.g `apimodels.AdminConfiguration` vs `ngmodels.AdminConfiguration`

Reply:
If the latter is redable, then I see no issues

Reply:
I've actually changed how this looks after doing the API models - WDYT?